source/intro.tex

```diff
@@ -424,8 +424,8 @@
 
 \pnum
 Throughout this International Standard, each example is introduced by
-``\enterexample'' and terminated by ``\exitexample''. Each note is
-introduced by ``\enternote'' and terminated by ``\exitnote''. Examples
+``\begin{example}'' and terminated by ``\end{example}''. Each note is
```

Comment:
This isn't a legitimate way to use environments.

(I have already fixed that in a separate branch.)